app.py

```diff
@@ -178,6 +247,10 @@ def main():
     }
     for protocol in protocols:
         protocol_main_chart_data = protocol_main_chart_data_mapping[protocol]
+        if protocol_main_chart_data is None:
+            streamlit.subheader(
+                f":warning: No data for STRK and {src.settings.STABLECOIN_BUNDLE_NAME} bundle")   
+            break
         if protocol_main_chart_data.empty:
             continue
```

Comment:
seems like data are missing only for nostra alpha protocol. If we break adding protocol data we might miss all the data from protocols that would appear after nostra alpha in this loop. 
Please combine this condition with one on line 254 as they have similar nature. Drop subheader as it is missleading - we have data for `STRK-<bundle>`, but this pair is not present on some protocols. Log warning instead, something like `No data for pair {debt_token} - {collateral_token} from {protocol}`

Reply:
done! ✅